### ISSUE TITLE
summarize_ds_models should check that truncations are all the same

### ISSUE THREAD

**Issue by dill** [DistanceDevelopment/Distance#16]
Otherwise AIC comparison is not valid!

**Comment by erex** [comment]
Likewise bins (at the minute doesn't work with binned data)

*see distance list correspondence 28Apr17 with Carolin Troger*

```
 I tried it with the minke data, and the function summarize_ds_model worked. So the problem must be within my data. 
Then i checked the summarize_ds_model with my data :

test<-ds(Wi16, key="hn", formula=~1, convert.units=0.001)
test2<-ds(So16, key="hr",formula=~1, convert.units=0.001)
summarize_ds_models(test, test2)

WORKS !!

BUT as soon as i put cut points in the ds formula, the error pops up :

test<-ds(Wi16, key="hn", formula=~1, convert.units=0.001)
test2<-ds(So16, key="hr",formula=~1, convert.units=0.001, cutpoints=dist.bins)
summarize_ds_models(test,test2)

Fehler in data.frame(c("Hazard-rate", "~1", "0.439498409297398", "0.0158987457444908",  : 
  arguments imply differing number of rows: 5, 6
```

**Comment by dill** [comment]
fixed in 52eb627
